ts/tat-trend.ts

```diff
@@ -96,10 +95,13 @@ function getCompletedDateAndDays(
       days = row[COLUMN_NAMES.FD_DAYS] ?? 0;
       break;
     case "Full Case":
-      completedDate = row[`${dataTypePrefix} Release Completed`]
-        ? new Date(row[`${dataTypePrefix} Release Completed`])
+      const completedKey =
+        `${dataTypePrefix}_COMPLETED` as keyof typeof COLUMN_NAMES;
+      const daysKey = `${dataTypePrefix}_DAYS` as keyof typeof COLUMN_NAMES;
+      completedDate = row[COLUMN_NAMES[completedKey]]
+        ? new Date(row[COLUMN_NAMES[completedKey]])
         : null;
-      days = row[`${dataTypePrefix} Total Days`] ?? 0;
+      days = row[COLUMN_NAMES[daysKey]] ?? 0;
```

Comment:
The column names are still being constructed here. Casting them as constant values doesn't help.

All strings that refer to column names from the Case TAT Report should be either direct references to the constants, or use a function to construct them. This way, it's all in one place and much easier to maintain. There should also be comments by the constants and any such functions to explain that they are used for referring to the Case TAT Report.